<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUsers</class>
 <widget class="QWidget" name="FormUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Users List</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidgetUsers"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonDelUser">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/img/fl.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNewUsername">
             <property name="text">
              <string>newuser</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNewPassword">
             <property name="text">
              <string>newpassword</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddUser">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Users Log</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
